### ISSUE TITLE
Error in missing rootfs.tar.gz

### ISSUE THREAD

**Issue by ubuntu2029** [JamiKettunen/initramfs-tools#4]

**Comment by ubuntu2029** [comment]
Hi, 
I followed your https://github.com/JamiKettunen/linux-mainline-oneplus5, i could compile the kernel. But unfortunately i could not create a initramfs for the device by  https://github.com/JamiKettunen/initramfs-tools.  
My purpose:
1), Compile a bootable kernel and create a boot.img for oneplus 5t
2), I want boot 5t into ubuntu gnome environment.

BUT BUT stuck in first step, i am a fresh learner.  Need your help and give some instructions. Many thanks.
git clone https://github.com/JamiKettunen/initramfs-tools.git
Cloning into 'initramfs-tools'...
remote: Enumerating objects: 381, done.
remote: Counting objects: 100% (381/381), done.
remote: Compressing objects: 100% (237/237), done.
remote: Total 381 (delta 191), reused 304 (delta 115), pack-reused 0
Receiving objects: 100% (381/381), 115.86 KiB | 775.00 KiB/s, done.
Resolving deltas: 100% (191/191), done.
root@ucompiler:~# cd buildroot-2021.02.9^C
root@ucompiler:~# cd initramfs-tools/
root@ucompiler:~/initramfs-tools# ./build.sh -N
Cloning into 'buildroot-git'...
remote: Enumerating objects: 8318, done.
remote: Counting objects: 100% (8318/8318), done.
remote: Compressing objects: 100% (4975/4975), done.
remote: Total 452884 (delta 4411), reused 6487 (delta 3314), pack-reused 444566
Receiving objects: 100% (452884/452884), 98.18 MiB | 4.31 MiB/s, done.
Resolving deltas: 100% (317139/317139), done.
>> Adding 2 out-of-tree patch(es)...
mkdir -p /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/lxdialog
PKG_CONFIG_PATH="" make CC="/usr/bin/gcc" HOSTCC="/usr/bin/gcc" \
    obj=/root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config -C support/kconfig -f Makefile.br conf
make[1]: Entering directory '/root/initramfs-tools/buildroot/buildroot-git/support/kconfig'
/usr/bin/gcc -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -DCURSES_LOC="<ncurses.h>" -DNCURSES_WIDECHAR=1 -DLOCALE  -I/root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config -DCONFIG_=\"\"  -MM *.c > /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/.depend 2>/dev/null || :
/usr/bin/gcc -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -DCURSES_LOC="<ncurses.h>" -DNCURSES_WIDECHAR=1 -DLOCALE  -I/root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config -DCONFIG_=\"\"   -c conf.c -o /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/conf.o
/usr/bin/gcc -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -DCURSES_LOC="<ncurses.h>" -DNCURSES_WIDECHAR=1 -DLOCALE  -I/root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config -DCONFIG_=\"\"  -I. -c /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/zconf.tab.c -o /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/zconf.tab.o
/usr/bin/gcc -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600 -DCURSES_LOC="<ncurses.h>" -DNCURSES_WIDECHAR=1 -DLOCALE  -I/root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config -DCONFIG_=\"\"   /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/conf.o /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/zconf.tab.o  -o /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/conf
rm /root/initramfs-tools/buildroot/buildroot-git/output/build/buildroot-config/zconf.tab.c
make[1]: Leaving directory '/root/initramfs-tools/buildroot/buildroot-git/support/kconfig'
#
# configuration written to /root/initramfs-tools/buildroot/buildroot-git/.config
#
>> Starting build with 12 jobs...
Makefile.legacy:9: *** "You have legacy configuration in your .config! Please check your configuration.".  Stop.

real	0m0.269s
user	0m0.183s
sys	0m0.153s

**Comment by JamiKettunen** [comment]
Should be fixed by 73c656e, but I also recommend you to get my latest mainline 5.17 tree from [msm8998-mainline/linux](https://gitlab.com/msm8998-mainline/linux) (I need to add a deprecation notice to the one on my GitHub at some point).

I've not yet pushed the config files to the tree unlike here so you can do something like this instead to make a `.config` for now on that newer kernel:
```bash
curl https://gitlab.com/postmarketOS/pmaports/-/raw/master/device/testing/linux-postmarketos-qcom-msm8998/config-postmarketos-qcom-msm8998.aarch64 -o arch/arm64/configs/pmos_defconfig
make ... pmos_defconfig
```
